### ISSUE TITLE
Feature request: ar_cross_targets()

### ISSUE THREAD

**Issue by samuelae** [samuelae/associatoR#18]
The \texttt{ar\_cross\_targets()} produces cross-tables between variables in the \texttt{targets} and \texttt{participants} tables while respecting the retrieval counts from \texttt{responses}. This can be used, for instance, to evaluate the proportion of responses from each target cluster for different individual difference factors. The function takes the \texttt{associatoR} object as input and returns a table of class \texttt{tibble} containing grouped statistics. Per default, the function produces frequencies, but the user can request group-normalized counts using the \texttt{normalize} argument.  

```
ar_cross_targets(ar_obj, 
                 participant_vars = c(gender, education), 
                 target_var = cluster, 
                 normalize = TRUE)
```

**Comment by samuelae** [comment]
current description: The function \texttt{ar\_cross\_targets()} generates cross-tables of frequencies of targets in responses between \texttt{targets} and \texttt{participants} variables. This function can be used, for instance, to evaluate the proportion of responses from each target cluster for different individual difference factors. The function takes the associatoR object as input and returns a table of grouped statistics. Per default, the function produces frequencies, but the user can request group-normalized counts using the \texttt{normalize} argument.

**Comment by samuelae** [comment]
implemented in commit 4466f75ef3662363627b6854aa01112fda3bf852
